src/components/Counter/Counter.tsx

```diff
@@ -27,13 +61,13 @@ const Counter = ({
   noAnimation
 }) => {
   // Variables
-  const overrideClassName = backwardCompatibilityForProperties([className, wrapperClassName]);
+  const overrideClassName = backwardCompatibilityForProperties([className, wrapperClassName], "") || undefined;
```

Comment:
Why was this needed?

Reply:
I have updated this now to return undefined from the `backwardCompatibilityForProperties` function directly as a default value. 

This was needed then because the empty string returned from the `backwardCompatibilityForProperties` call return
 would cause the addition of a redundant `className=""` prop to the output element. By using undefined as the value instead the prop is not added to the element.

Reply:
Got it 👍